package.json

```diff
@@ -96,6 +96,7 @@
     "resource-router-middleware": "^0.6.0",
     "sharp": "^0.23.4",
     "soap": "^0.25.0",
+    "storefront-query-builder": "https://github.com/DivanteLtd/storefront-query-builder.git",
```

Comment:
Here I have a small question. 
Why is it not released with a version and pulled with that?

Reply:
that's actually a good idea but how to mark the version with github url? :D

Reply:
You would do it like this.
https://github.com/DivanteLtd/storefront-query-builder.git#<version\>
OR
https://github.com/DivanteLtd/storefront-query-builder.git#semver:<version\>

But I would more say that it should be published to npm in the long run.